Gtk window contents not rendering on MacOS (Mojave)

Is the error we're seeing here a bug or misuse issue via `VideoIO` or `ImageView`?
https://github.com/JuliaIO/VideoIO.jl/pull/150

```julia
julia> VideoIO.viewcam()
FATAL ERROR: Gtk state corrupted by error thrown in a callback:
ERROR: AssertionError: g_stack === nothing && !prev
Stacktrace:
 [1] g_sigatom(::Any) at /Users/ian/.julia/packages/Gtk/aP55V/src/GLib/signals.jl:165
 [2] macro expansion at /Users/ian/.julia/packages/Gtk/aP55V/src/GLib/signals.jl:191 [inlined]
 [3] show at /Users/ian/.julia/packages/Gtk/aP55V/src/base.jl:35 [inlined]
 [4] Gtk.GtkWindowLeaf(::String, ::Int64, ::Int64, ::Bool, ::Bool) at /Users/ian/.julia/packages/Gtk/aP55V/src/windows.jl:14
 [5] Type at /Users/ian/.julia/packages/Gtk/aP55V/src/windows.jl:2 [inlined]
 [6] #GtkWindow#188 at /Users/ian/.julia/packages/Gtk/aP55V/src/GLib/gtype.jl:226 [inlined]
 [7] Type at /Users/ian/.julia/packages/Gtk/aP55V/src/GLib/gtype.jl:226 [inlined]
 [8] #imshow_gui#26(::String, ::Symbol, ::Function, ::Tuple{Int64,Int64}, ::ImageView.SliceData{false,0,Tuple{}}, ::Tuple{Int64,Int64}) at /Users/ian/.julia/packages/ImageView/1uiRS/src/ImageView.jl:295
 [9] #imshow_gui at ./none:0 [inlined] (repeats 2 times)
 [10] #imshow#17(::String, ::Symbol, ::Function, ::SubArray{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2,MappedArrays.ReadonlyMappedArray{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2,PermutedDimsArray{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2,(2, 1),(2, 1),Array{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2}},typeof(identity)},Tuple{Base.OneTo{Int64},StepRange{Int64,Int64}},false}, ::Reactive.Signal{CLim{ColorTypes.RGB{Float64}}}, ::Reactive.Signal{GtkReactive.ZoomRegion{RoundingIntegers.RInt64}}, ::ImageView.SliceData{false,0,Tuple{}}, ::Reactive.Signal{Dict{UInt64,Any}}) at /Users/ian/.julia/packages/ImageView/1uiRS/src/ImageView.jl:200
 [11] #imshow at ./none:0 [inlined] (repeats 2 times)
 [12] #imshow#13(::Tuple{Int64,Int64}, ::String, ::Function, ::Symbol, ::Base.Iterators.Pairs{Symbol,Bool,Tuple{Symbol,Symbol},NamedTuple{(:flipx, :interactive),Tuple{Bool,Bool}}}, ::Function, ::PermutedDimsArray{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2,(2, 1),(2, 1),Array{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2}}) at /Users/ian/.julia/packages/ImageView/1uiRS/src/ImageView.jl:177
 [13] (::getfield(ImageView, Symbol("#kw##imshow")))(::NamedTuple{(:flipx, :interactive),Tuple{Bool,Bool}}, ::typeof(imshow), ::PermutedDimsArray{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2,(2, 1),(2, 1),Array{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2}}) at ./none:0
 [14] play(::VideoIO.VideoReader{true}, ::Bool) at /Users/ian/Documents/GitHub/VideoIO.jl/src/VideoIO.jl:84
 [15] viewcam(::String, ::Ptr{VideoIO.AVFormat.AVInputFormat}) at /Users/ian/Documents/GitHub/VideoIO.jl/src/VideoIO.jl:101
 [16] viewcam() at /Users/ian/Documents/GitHub/VideoIO.jl/src/VideoIO.jl:100
 [17] top-level scope at none:0

ERROR: AssertionError: g_stack === nothing && !prev
Stacktrace:
 [1] g_sigatom(::Any) at /Users/ian/.julia/packages/Gtk/aP55V/src/GLib/signals.jl:165
 [2] macro expansion at /Users/ian/.julia/packages/Gtk/aP55V/src/GLib/signals.jl:191 [inlined]
 [3] show at /Users/ian/.julia/packages/Gtk/aP55V/src/base.jl:35 [inlined]
 [4] Gtk.GtkWindowLeaf(::String, ::Int64, ::Int64, ::Bool, ::Bool) at /Users/ian/.julia/packages/Gtk/aP55V/src/windows.jl:14
 [5] Type at /Users/ian/.julia/packages/Gtk/aP55V/src/windows.jl:2 [inlined]
 [6] #GtkWindow#188 at /Users/ian/.julia/packages/Gtk/aP55V/src/GLib/gtype.jl:226 [inlined]
 [7] Type at /Users/ian/.julia/packages/Gtk/aP55V/src/GLib/gtype.jl:226 [inlined]
 [8] #imshow_gui#26(::String, ::Symbol, ::Function, ::Tuple{Int64,Int64}, ::ImageView.SliceData{false,0,Tuple{}}, ::Tuple{Int64,Int64}) at /Users/ian/.julia/packages/ImageView/1uiRS/src/ImageView.jl:295
 [9] #imshow_gui at ./none:0 [inlined] (repeats 2 times)
 [10] #imshow#17(::String, ::Symbol, ::Function, ::SubArray{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2,MappedArrays.ReadonlyMappedArray{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2,PermutedDimsArray{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2,(2, 1),(2, 1),Array{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2}},typeof(identity)},Tuple{Base.OneTo{Int64},StepRange{Int64,Int64}},false}, ::Reactive.Signal{CLim{ColorTypes.RGB{Float64}}}, ::Reactive.Signal{GtkReactive.ZoomRegion{RoundingIntegers.RInt64}}, ::ImageView.SliceData{false,0,Tuple{}}, ::Reactive.Signal{Dict{UInt64,Any}}) at /Users/ian/.julia/packages/ImageView/1uiRS/src/ImageView.jl:200
 [11] #imshow at ./none:0 [inlined] (repeats 2 times)
 [12] #imshow#13(::Tuple{Int64,Int64}, ::String, ::Function, ::Symbol, ::Base.Iterators.Pairs{Symbol,Bool,Tuple{Symbol,Symbol},NamedTuple{(:flipx, :interactive),Tuple{Bool,Bool}}}, ::Function, ::PermutedDimsArray{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2,(2, 1),(2, 1),Array{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2}}) at /Users/ian/.julia/packages/ImageView/1uiRS/src/ImageView.jl:177
 [13] (::getfield(ImageView, Symbol("#kw##imshow")))(::NamedTuple{(:flipx, :interactive),Tuple{Bool,Bool}}, ::typeof(imshow), ::PermutedDimsArray{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2,(2, 1),(2, 1),Array{ColorTypes.RGB{FixedPointNumbers.Normed{UInt8,8}},2}}) at ./none:0
 [14] play(::VideoIO.VideoReader{true}, ::Bool) at /Users/ian/Documents/GitHub/VideoIO.jl/src/VideoIO.jl:84
 [15] viewcam(::String, ::Ptr{VideoIO.AVFormat.AVInputFormat}) at /Users/ian/Documents/GitHub/VideoIO.jl/src/VideoIO.jl:101
 [16] viewcam() at /Users/ian/Documents/GitHub/VideoIO.jl/src/VideoIO.jl:100
 [17] top-level scope at none:0
```

@kmsquire

In search of reasons... It looks like my `gtk+3` brew install is broken.

```julia
julia> Homebrew.brew(`reinstall gtk+3`)
==> Reinstalling gtk+3 
==> Downloading https://download.gnome.org/sources/gtk+/3.24/gtk+-3.24.8.tar.xz
Already downloaded: /Users/ian/Library/Caches/Homebrew.jl/downloads/e962619b2823e9ecdba6611331876cf0daa34bebcfaa921eb61d03d300b1c22b--gtk -3.24.8.tar.xz
==> ./configure --enable-debug=minimal --prefix=/Users/ian/.julia/packages/Homebrew/s09IX/deps/usr/Cellar/gtk+3/3.24.8 --disable-glibtest --enable-introspection=yes --disable-schemas-compile --enable-qua
==> make install
Last 15 lines from /Users/ian/Library/Logs/Homebrew/gtk+3/02.make:
  CC       libgdk_3_la-gdkenumtypes.lo
  CC       libgdk_3_la-gdkmarshalers.lo
  CC       libgdk_3_la-gdkresources.lo
  GEN      gdkconfig.h
  CC       deprecated/libgdk_3_la-gdkcolor.lo
  CCLD     libgdk-3.la
  GISCAN   Gdk-3.0.gir
gdkversionmacros.h:35: Warning: Gdk: multiple comment blocks documenting 'GDK_DISABLE_DEPRECATION_WARNINGS:' identifier (already seen at gdk.c:106).
  GICOMP   Gdk-3.0.gir
Could not find GIR file 'GdkPixbuf-2.0.gir'; check XDG_DATA_DIRS or use --includedir
error parsing file Gdk-3.0.gir: Failed to parse included gir GdkPixbuf-2.0
make[3]: *** [Gdk-3.0.typelib] Error 1
make[2]: *** [install-recursive] Error 1
make[1]: *** [install] Error 2
make: *** [install-recursive] Error 1

READ THIS: https://docs.brew.sh/Troubleshooting

ERROR: failed process: Process(`/Users/ian/.julia/packages/Homebrew/s09IX/deps/usr/bin/brew reinstall gtk+3`, ProcessExited(1)) [1]
Stacktrace:
 [1] error(::String, ::Base.Process, ::String, ::Int64, ::String) at ./error.jl:42
 [2] pipeline_error at ./process.jl:785 [inlined]
 [3] #run#515(::Bool, ::Function, ::Cmd) at ./process.jl:726
 [4] run at ./process.jl:724 [inlined]
 [5] #brew#4(::Bool, ::Bool, ::Bool, ::Bool, ::Bool, ::Function, ::Cmd) at /Users/ian/.julia/packages/Homebrew/s09IX/src/API.jl:19
 [6] brew(::Cmd) at /Users/ian/.julia/packages/Homebrew/s09IX/src/API.jl:11
 [7] top-level scope at none:0
```
Running ```Homebrew.brew(`reinstall gdk-pixbuf`)``` finishes successfully, but doesn't fix the above

--- a/docs/src/index.md
+++ b/docs/src/index.md
@@ -30,3 +30,14 @@ Gtk is precompilable by normal mechanisms. For further reduction of startup time
 ## Attribution
 
 Gtk logo is made by Andreas Nilsson [[GFDL](https://www.gnu.org/copyleft/fdl.html) or [CC-BY-SA-3.0](https://creativecommons.org/licenses/by-sa/3.0/)], via Wikimedia Commons
+
+## Common Issues
+
+If you are running Gtk on MacOS Mojave, it's likely that when you try one of te examples, a blank window will render. In order to fix this, you can run the following commands which will downgrade your version of `glib` to the compatible version for `Gtk.jl`
+
+Command to run: 
+```julia
+using Homebrew
+Homebrew.brew(`unlink glib`)
+Homebrew.brew(`install https://raw.githubusercontent.com/Homebrew/homebrew-core/b27a055812fe620e0d3dbe67f2a424ed3a846ecf/Formula/glib.rb`)
+```

